server/asset/dcr/dcr.go

```diff
@@ -559,3 +557,8 @@ func toCoinID(txHash *chainhash.Hash, vout uint32) []byte {
 	binary.BigEndian.PutUint32(b[hashLen:], vout)
 	return b
 }
+
+// Convert the DCR value to atoms.
+func toAtoms(v float64) uint64 {
+	return uint64(math.Round(v * 1e8))
```

Comment:
I'm always squeamish about multiplying in float, but this is what dcrutil.NewAmount does so I suppose it's sufficient for a *1e8.